### PR TITLE
WN Update: SignalR ActivitySource

### LINKED ISSUE #33546: WN: Add SignalR distributed tracing section and code sample
1. Add the new section for SignalR distributed tracing based on the dotnet/release-notes-draft.
2. Edit sweep
3. If any code samples, then move inline code into a runnable project and link the code back in.

---
[Associated WorkItem - 317469](https://dev.azure.com/msft-skilling/Content/_workitems/edit/317469)

### PR DESCRIPTION
**Pull request by wadepickett**
[Internal preview of updated SignalR section for Improved Activities](https://review.learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-9.0?view=aspnetcore-9.0&branch=pr-en-us-34183#improved-activities-for-signalr)

Added a missing new activity source for SignalR.

Fixes #33546

Why:
I had previously added a new include file with the update many weeks ago which was never referenced in the What's New, rather than updating the original include. Fixed that so the update is in the original include instead, and removed the duplicate include.

Also:
- Moved the inline code example out to an app sample in a program.cs and linked back in.
- Updated the ms.date in the What's new topic.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/release-notes/aspnetcore-9.0.md](https://github.com/dotnet/AspNetCore.Docs/blob/3fda76e5bb1d5eada88392ea5aac4f09a343b5e0/aspnetcore/release-notes/aspnetcore-9.0.md) | [aspnetcore/release-notes/aspnetcore-9.0](https://review.learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-9.0?branch=pr-en-us-34183) |
| [aspnetcore/release-notes/aspnetcore-9/samples/SignalRChatTraceExample/wwwroot/lib/jquery-validation/LICENSE.md](https://github.com/dotnet/AspNetCore.Docs/blob/3fda76e5bb1d5eada88392ea5aac4f09a343b5e0/aspnetcore/release-notes/aspnetcore-9/samples/SignalRChatTraceExample/wwwroot/lib/jquery-validation/LICENSE.md) | [aspnetcore/release-notes/aspnetcore-9/samples/SignalRChatTraceExample/wwwroot/lib/jquery-validation/LICENSE](https://review.learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-9/samples/SignalRChatTraceExample/wwwroot/lib/jquery-validation/LICENSE?branch=pr-en-us-34183) |

### CODE REVIEW

**Comment by wadepickett** [comment]
@BrennanConroy, [Internal preview of updated SignalR section for Improved Activities](https://review.learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-9.0?view=aspnetcore-9.0&branch=pr-en-us-34183#improved-activities-for-signalr)

File: aspnetcore/release-notes/aspnetcore-9/includes/signalrActivities.md [hunk 1]
```diff
@@ -15,29 +19,19 @@ The following example uses the [.NET Aspire dashboard](/dotnet/aspire/fundamenta
...
+#### .NET SignalR client ActivitySource
 
-        tracing.AddAspNetCoreInstrumentation();
-        tracing.AddSource("Microsoft.AspNetCore.SignalR.Server");
-    });
+The SignalR ActivitySource named `Microsoft.AspNetCore.SignalR.Client` emits events for a SignalR client:
 
-builder.Services.ConfigureOpenTelemetryTracerProvider(tracing => tracing.AddOtlpExporter());
-```
+* .NET SignalR client has an `ActivitySource` named `Microsoft.AspNetCore.SignalR.Client`. Hub invocations now create a client span. Note that other SignalR clients, such as the JavaScript client, don't support tracing. This feature will be added to more clients in future releases.
```
Comment (tdykstra) [inline aspnetcore/release-notes/aspnetcore-9/includes/signalrActivities.md:32]:
```suggestion
* The .NET SignalR client has an `ActivitySource` named `Microsoft.AspNetCore.SignalR.Client`. Hub invocations now create a client span. Note that other SignalR clients, such as the JavaScript client, don't support tracing. This feature will be added to more clients in future releases.
```

**Review by tdykstra — APPROVED** [review]
LGTM. One nit suggestion.

File: aspnetcore/release-notes/aspnetcore-9/samples/SignalRChatTraceExample/Program.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+using OpenTelemetry.Trace;
+using SignalRChat.Hubs;
+
+// Set OTEL_EXPORTER_OTLP_ENDPOINT environment variable depending on where your OTEL endpoint is.
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddRazorPages();
+builder.Services.AddSignalR();
+
+builder.Services.AddRazorPages();
```
Comment (BrennanConroy) [inline aspnetcore/release-notes/aspnetcore-9/samples/SignalRChatTraceExample/Program.cs:11]: Remove duplicate code

Comment (wadepickett) [reply aspnetcore/release-notes/aspnetcore-9/samples/SignalRChatTraceExample/Program.cs:11]: Thanks for the catch, Brennan, sorry I missed that.  Correcting....

**Pull request merged**